modules/vstudio/vs200x_vcproj.lua

```diff
@@ -819,10 +819,54 @@
 		end
 	end
 
-
+	function tablefindandremove(tab, el)
+		for index, value in pairs(tab) do
+			if value == el then
+				table.remove(tab, index)
+				return true
+			end
+		end
+		return false
+	end
 
 	function m.additionalExternalCompilerOptions(cfg, toolset)
 		local buildoptions = table.join(toolset.getcxxflags(cfg), cfg.buildoptions)
```

Comment:
I think a simpler solution would be to replace this with `local buildoptions = cfg.buildoptions`.
Them pulling only the wanted options from the toolset and having functions called to output the correct tags like the vs2010 generator.